### DISCUSSION TITLE
How to hide parent folder on winbar?

### DISCUSSION THREAD

**Discussion by fnacarellidev** [nvimdev/lspsaga.nvim#808] [Q&A]
This is my current setup for symbol_in_winbar module (default)

![imagem_2023-01-31_225038243](https://user-images.githubusercontent.com/97247063/215925362-ce0c9c9a-4b51-482f-a321-de2c64f9b78a.png)

and it displays this horrendous thing at the top, I don't want the parent folder, only the name of the file that I am currently editing, how do I do this?

![imagem_2023-01-31_225143707](https://user-images.githubusercontent.com/97247063/215925498-8c36eb21-3b11-43f6-a1cb-56d0dad0bb51.png)

**Comment by glepnir** [comment]
folder level to1.

**Reply by fnacarellidev** [reply]
Already tried it, works on some files, doesn't work on others files, for example, in this C file it works as 

![imagem_2023-01-31_225835011](https://user-images.githubusercontent.com/97247063/215926385-7a0fe315-d8e4-4dc0-8a6c-004a68ca79da.png)

but in the .lua file it doesn't.

![image](https://user-images.githubusercontent.com/97247063/215926428-6cf31006-3111-42fe-917a-3bef0cdea268.png)



![imagem_2023-01-31_225747472](https://user-images.githubusercontent.com/97247063/215926281-7d9a5438-35f9-45a0-a9e5-ad0d0a9a15f7.png)

**Reply by glepnir** [reply]
maybe a bug test it later

**Reply by glepnir** [reply]
lua file has root dir? in lsp info

**Reply by fnacarellidev** [reply]
Ran `:Lsp info`, and it tells "root directory: Running in single file mode."

**Reply by glepnir** [reply]
lua file path is what looks like no root dir

**Reply by fnacarellidev** [reply]
What should I do?

**Reply by glepnir** [reply]
maybe i know where is bug. fix it later in gym now

**Reply by fnacarellidev** [reply]
thanks!

**Comment by glepnir** [comment]
wired just a test. it works fine. 

![image](https://user-images.githubusercontent.com/41671631/216184907-625f0669-4b4d-43e6-a411-bac42d2bed79.png)

**Reply by fnacarellidev** [reply]
Alright, do you know how I can debug and try to find where is the problem?

**Reply by glepnir** [reply]
`:echo bufname()` output is what.
